src/duke/DukeException/DukeException.java

```diff
@@ -0,0 +1,9 @@
+package duke.DukeException;
+
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
+public class DukeException extends Exception {
+
+
+}
```

Comment:
Can consider shifting InsufficientArgumentException and InvalidCommandException here instead of creating more empty classes with no code